backend/tests/unit_tests/config/test_update.py

```diff
@@ -54,14 +53,16 @@ def _run_u_config(
             data,
             schema,
             _,
+            expected_error,
```

Comment:
Since we are using `.get('expected_error')` in the payload extractor, setting it as `null` is not important. A positive test case can be inferred if the payload does not have they key. It's straightforward.